### PR TITLE
decoration: Fix flat-brace rotation

### LINKED ISSUE #519: There might be a bug while using `flat-brace` with `anchor`.
There might be a bug while using `flat-brace` with `anchor`.

For example, I expected to get a `flat-brace` pointing to point O and B (whose anchor is defined in the plot), but the direction seems to be wrong.
PixPin_2024-03-02_10-15-22

Same code with `brace` won't produce this issue.
PixPin_2024-03-02_10-15-43

Code in pictures:
```
#let c=3
#cetz.canvas(length: 2.5cm, 
{
  import cetz.draw: *
  import cetz.plot

  plot.plot(
    size: (2.6, 2.6),
    axis-style: "school-book",

    x-max: 8.5,
    x-min: -1.5,
    y-max: 8.5,
    y-min: -1.5,

    x-tick-step: 5,
    x-minor-tick-step: 1,
    y-tick-step: 5,
    y-minor-tick-step: 1,

    x-grid: "both",
    y-grid: "both",

    name: "p",
    {
      plot.add(
        axes: ("x", "y"),
        
        domain: (-2, 11), x => (x * x - c * c)/(2 * c)
      )
      plot.add(
        axes: ("x", "y"),
        domain: (-2, 11), x => -1 / 2 *  (x - 6) + 4.5
      )
      plot.add(
        axes: ("x", "y"),
        domain: (0, 6), x => 4.5 / 6 * x
      )

      plot.add-anchor("O", (0, 0))
      plot.add-anchor("A", (6, 4.5))
      plot.add-anchor("B", (0, 7.5))
    }
  )
  
  circle("p.O", radius: 0.03, name: "pointO")
  content((rel: (-0.1, 0.1)), $O$)

  circle("p.A", radius: 0.03)
  content((rel: (0.1, 0.1)), $A$)

  circle("p.B", radius: 0.03)
  content((rel: (0.1, 0.1)), $B$)

  cetz.decorations.brace("p.B", "p.O")

})
```

_Originally posted by @ParaN3xus in https://github.com/johannes-wolf/cetz/issues/238#issuecomment-1974202555_

### PR DESCRIPTION
**Pull request by johannes-wolf**
Fixes #519.

### CODE REVIEW

File: src/lib/decorations/brace.typ [hunk 1]
```diff
@@ -321,7 +321,7 @@
     // pointing upwards, so we set the origin and rotate the entire group accordingly
     let (_, start, end) = coordinate.resolve(ctx, start, end)
     set-origin(start)
-    rotate(vector.angle2(start, end) * -1)
+    rotate(vector.angle2(start, end))
```
Comment (fenjalien) [inline src/lib/decorations/brace.typ:324]: Can you add a comment about why it needs to be multiplied by -1?

**Review by fenjalien — COMMENTED** [review]
Changelog!

File: src/lib/decorations/brace.typ (hunk 1 above)
Comment (johannes-wolf) [reply src/lib/decorations/brace.typ:324]: It does not. I've removed the `* -1`.

Comment (fenjalien) [reply src/lib/decorations/brace.typ:324]: *OH* my bad

**Review by fenjalien — APPROVED** [review]

**Pull request merged**